### PR TITLE
mark workspace and deployment creation optional in mwaa and gcc migration docs

### PR DESCRIPTION
**Pull request by melugoyal**
marking those steps as optional because customers may already have a workspace and deployment

### CODE REVIEW

**Review by lzdanski — COMMENTED** [review]
Looking at the content of these steps, I think they do need a sentence in the text body explaining what about the step is optional ie, you need to have a workspace and organization to migrate, but you don't have to make new ones specifically for it. These docs use partials though, so the content is in a different file. I can make the change and get your approval on it.

File: astro/migration-partials/deployments.mdx [hunk 1]
```diff
@@ -3,6 +3,8 @@ import TabItem from "@theme/TabItem";
 
 A _Deployment_ is an Astro Runtime environment that is powered by the core components of Apache Airflow. In a Deployment, you can deploy and run DAGs, configure worker resources, and view metrics.
 
+You can choose to use an existing Deployment, or create a new one. But you must have at least one Deployment to complete your migration.
```
Comment (lzdanski) [inline astro/migration-partials/deployments.mdx:6]: @melugoyal - how do you feel about us including this brief explanation with this change?

Comment (melugoyal) [reply astro/migration-partials/deployments.mdx:6]: works for me, thanks

Comment (lzdanski) [inline astro/migration-partials/deployments.mdx:6]:
```suggestion
You can choose to use an existing Deployment, or create a new one. However, you must have at least one Deployment to complete your migration.
```

File: astro/migration-partials/workspaces.mdx [hunk 2]
```diff
@@ -1,4 +1,6 @@
-In your Astro Organization, you can create _Workspaces_, which are a collection of users that have access to the same Deployments. Workspaces are typically owned by a single team. 
+In your Astro Organization, you can create _Workspaces_, which are a collection of users that have access to the same Deployments. Workspaces are typically owned by a single team.
+
+You can choose to use an existing Workspace, or create a new one. But you must have at least one Workspace to complete your migration.
```
Comment (lzdanski) [inline astro/migration-partials/workspaces.mdx:3]:
```suggestion
You can choose to use an existing Workspace, or create a new one. However, you must have at least one Workspace to complete your migration.
```

**Review by lzdanski — APPROVED** [review]

**Pull request merged**